src/xdg.rs

```diff
@@ -329,6 +329,31 @@ impl BaseDirectories
         list_files(self.get_runtime_directory(), &Vec::new(),
                    self.prefix.join(path))
     }
+
+    /// Returns the `XDG_DATA_HOME` directory.
```

Comment:
Since these functions never rethrn an `Option`, I think a more clear phrasing of the docstring would be:
```
/// Returns the user-specific data directory
/// (set by `XDG_DATA_HOME` environment variable).
```
and so on.